cluster/src/main/java/org/apache/iotdb/cluster/server/member/RaftMember.java

```diff
@@ -260,6 +260,7 @@ public void start() {
     }
 
     startBackGroundThreads();
+    setSkipElection(false);
```

Comment:
setSkipElection(false);  放到 startBackGroundThreads(); 前面， 是不是 更好？

Reply:
If so, the status will be 'ready to launch election' before all the necessary threads starting (You can reference `RaftMember.startBackGroundThreads`). There are some corner cases that the heartbeat thread starts and launchs the election, but other threads, like `commitLog` and `appendLog` is not ready yet. Some unknown status and error may occur.
The safe way is to let `start()` be just a reverse-mirror of `stop()`.

Reply:
If because of this concern, what will happen when first time running this startBackGroundThreads() after IoTDB process starting up?
At that time, the skipElection is just false before running startBackGroundThreads().

Reply:
You are right! The default value should be `true`, not `false`. I will optimize this in a new PR.